cve_bin_tool/output_engine/html_reports/templates/row_product.html

```diff
@@ -79,25 +72,21 @@ <h6><strong>Severity</strong></h6>
                                                 </div>
                                             </div>
                                         </div>
-                                        {{ new_cves }}
-                                        {% if confirmed_cves %}
-                                        <span id="remark-confirmed{{ fix_id}}"></span>
-                                        {{ confirmed_cves }}
-                                        {%endif%}
-                                        {% if mitigated_cves %}
-                                        <span id="remark-mitigated{{ fix_id}}"></span>
-                                        {{ mitigated_cves }}
-                                        {%endif%}
-
-                                        {% if unexplored_cves %}
-                                        <span id="remark-unexplored{{ fix_id}}"></span>
-                                        {{ unexplored_cves }}
-                                        {%endif%}
-
-                                        {% if ignored_cves %}
-                                        <span id="remark-ignored{{ fix_id}}"></span>
-                                        {{ ignored_cves }}
-                                        {%endif%}
+                                        <div class="new">
```

Comment:
Should we be taking out the ifs here?

Reply:
if we conditionally render those div, javascript will throw error while filtering, because it will look for those classes to apply or remove property `display: none`